src/js/components/FormGroupComponent.jsx

```diff
@@ -33,7 +35,7 @@ var FormGroupComponent = React.createClass({
 
     return (
       <div className="help-block">
-        <strong>{props.errorMessage}</strong>
+        <AutolinkComponent text={props.errorMessage} />
```

Comment:
Yeah i thought about it, but I kinda wanted to explicitly make the prop be a String rather than a Node (children must be). I also wanted to keep it close to the implementation of the package I took the inspiration from so when we update to React 0.14 it's just a matter of kicking this guy out.